tests/unselect_rules_list

```diff
@@ -3,23 +3,11 @@ xccdf_org.ssgproject.content_rule_ensure_gpgcheck_local_packages
 xccdf_org.ssgproject.content_rule_ensure_gpgcheck_never_disabled
 xccdf_org.ssgproject.content_rule_ensure_gpgcheck_repo_metadata
 xccdf_org.ssgproject.content_rule_ensure_redhat_gpgkey_installed
-xccdf_org.ssgproject.content_rule_rpm_verify_hashes
-xccdf_org.ssgproject.content_rule_rpm_verify_ownership
-xccdf_org.ssgproject.content_rule_rpm_verify_permissions
-xccdf_org.ssgproject.content_rule_grub2_enable_fips_mode
-xccdf_org.ssgproject.content_rule_grub2_password
-xccdf_org.ssgproject.content_rule_grub2_uefi_password
-xccdf_org.ssgproject.content_rule_installed_OS_is_certified
-xccdf_org.ssgproject.content_rule_install_antivirus
-xccdf_org.ssgproject.content_rule_install_hids
-xccdf_org.ssgproject.content_rule_install_mcafee_antivirus
-xccdf_org.ssgproject.content_rule_install_mcafee_cma_rt
-xccdf_org.ssgproject.content_rule_install_mcafee_hbss_accm
-xccdf_org.ssgproject.content_rule_install_mcafee_hbss_hips
-xccdf_org.ssgproject.content_rule_install_mcafee_hbss_pa
-xccdf_org.ssgproject.content_rule_mcafee_antivirus_definitions_updated
```

Comment:
@matusmarhefka Was the removal os this rules expected as well? They are not mentined in PR description.

Reply:
@yuumasato Yes it was, these rules have no remediation, therefore they can remain selected.

Reply:
@matusmarhefka The rules at the top of the chunk have remediation, like the ones related to `rule_rpm_verify_hashes`, permissions and ownership. Should they be removed as well?

Reply:
Yes, there should be only rules which might break the access to the machine for the test suite or destructive rules (which influence other rules remediations). We want to test rpm_verify family of rules. Of course, we can update this list in the future, so if you think that some rules should be added/removed from the list, please open a PR and we can discuss there.

Reply:
Just wanted to be sure. :)